config/README.md

```diff
@@ -1,102 +1,103 @@
-    docker build -f cmds/grpc-backend/Dockerfile  . -t gcr.io/$CLOUD_PROJECT/grpc-backend:$VERSION
-
-Push your new image to Google Container Registry:
-
-    docker push gcr.io/$CLOUD_PROJECT/http-gateway:$VERSION
-    docker push gcr.io/$CLOUD_PROJECT/grpc-backend:$VERSION
+Set the environment variable `DOCKER_URL` to your docker registry url endpoint.
 
 Use the `build.sh` script in this directory to build and push an image tagged
 with the current date and git commit hash.
 
 
 ## Creating a new Kubernetes cluster
 
-You need to create a Kubernetes cluster to run the cockroachdb instance and the
-gRPC and HTTP servers.  You can do this on any supported
-[cloud provider](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/)
-or even on your own infrastructure.  Instructions for GCP are given below,
-consult the Kubernetes documentation for other providers.
+This secion discusses deploying a kubernetes service, although you can deploy 
+a DSS node however you like as long as it meets the above requirements. You can
+do this on any supported [cloud provider](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/)
+or even on your own infrastructure.  Consult the Kubernetes documentation for
+which ever provider you choose.
 
-### Google Container Engine
 
-Create a new cluster in the given zone:
+### Deploying with Kubernetes
 
-    gcloud --project $CLOUD_PROJECT container clusters create $CLUSTER_NAME --zone $ZONE
-
-Fetch credentials for the cluster.  This populates your \~/.kube/config file
-and makes all future kubecfg commands target this cluster.
-
-    gcloud --project $CLOUD_PROJECT container clusters get-credentials $CLUSTER_NAME --zone $ZONE
-
-
-## Creating a new cockroachdb cluster
-
-1.  Create 5 static IP addresses.  How you do this depends on your cloud
-    provider.  3 IPs will be used for the 3 individual cockroachdb nodes, 1 IP
-    will be used to load balance amongst all the cockroachdb nodes, and 1 IP
-    will be used for the HTTPS frontend.  The HTTPS frontend IP will be used on
-    a Kubernetes Ingress, and if you're using Google Cloud it needs to be
-    created as a "Global" IP address.
+1.  Create your static IP addresses: 1 for each of your CockroachDB nodes (min
+    of 3) and 1 for the HTTPS Gateway's Ingress.  How you do this depends on
+    your cloud provider. Note: if you're using Google Cloud the HTTPS gateway
+    Ingress needs to be created as a "Global" IP address.
 
 1.  Copy `values.yaml.template` to `values.yaml` and fill in the required fields
     at the top.
 
 1.  Use the `make-certs.py` script in this directory to create certificates for
     the new cockroachdb cluster:
 
-        ./make-certs.py $NAMESPACE \
-            [--node-address <ADDRESS> <ADDRESS> <ADDRESS> ...]
-            [--ca-certs-dir <CA_CERTS_DIR>]
+        ./make-certs.py [--node-address <ADDRESS> <ADDRESS> <ADDRESS> ...]
+            [--ca-cert-to-join <CA_CERT_FILE>]
```

Comment:
This is in the "Creating a new CockroachDB cluster" section; how is the CA_CERT_FILE generated/obtained if not joining an existing cluster?

Reply:
args in brackets are optional, so if not joining a cluster you wouldn't specify it at all

Reply:
Let's make it clear in the sub-bullet below that --ca-cert-to-join should be omitted if not joining an existing cluster.

Under what conditions would the user not specify --node-address?

Reply:
done

Reply:
"Make sure you supply the following flags if joining an existing cluster".. i tried making this as clear as possible but open to suggestions if you have another way of wording it.

It's only required if joining an existing cluster, otherwise you omit it

Reply:
How about "Supply the following flags only if joining an existing cluster"?

But is that actually true?  If I am creating a cluster and I don't specify the --node-address flag, how will the addresses of my nodes be included in the node certificates?  And will another party be able to join my cluster later if I don't specify the addresses of my nodes in my nodes' certificates?

Reply:
done. yes, it uses service names for internal discovery

Reply:
I think we may want to take another look at this section in the future regarding which node addresses need to be specified when, but I think that can be addressed outside this PR.